### PR TITLE
In order to make this lib compatible with PHP 7.3, update the dependency to helum/config

### PR DESCRIPTION
**Pull request by Konafets**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "php": ">=7.0 <7.4",
         "ext-json": "*",
         "typo3/cms-core": "^7.6 || ^8.7",
-        "helhum/config-loader": "^0.9",
+        "helhum/config-loader": "^0.12",
```
Comment (helhum) [inline composer.json]: That would be a very restrictive constraint. Please use the same constraint as in master branch: https://github.com/helhum/typo3-config-handling/blob/3b8b4f8eb7b7b0ece7b20c5b79c7994d6056ba29/composer.json#L8

**Comment by helhum** [comment]
Good catch, thanks

File: composer.json (hunk 1 above)
Comment (Konafets) [reply composer.json]: Done.

**Comment by helhum** [comment]
released as 0.7.3

**Pull request merged**